src/honeybadger/authentic_handler.cr

```diff
@@ -0,0 +1,23 @@
+# The `Honeybadger::AuthenticHandler` is meant to integrate with the [Lucky web framework](https://luckyframework.org).
+#
+# It will attempt to fetch a user's ID from the current session, and include it in the error context if available.
+class Honeybadger::AuthenticHandler
+  include HTTP::Handler
+
+  def initialize(@factory : Honeybadger::HttpPayload.class = Honeybadger::HttpPayload)
+  end
+
+  def call(context : HTTP::Server::Context)
+    if (user_id = context.session.get?(Authentic::ActionHelpers::SIGN_IN_KEY))
```

Comment:
Let's add this session key to the initializer so it's more extensible.

Reply:
@stephendolan @robacarp does this still need work? lmk when you want me to merge. 👍

Reply:
Yeah, I still need to add tests 😄 

Currently refactoring my house in my spare time doing some renovations, so it may be a minute!